src/index.js

```diff
@@ -239,16 +239,21 @@ exports.init = (Implementation) => {
       const collections = _.values(Schemas.schemas);
       configStore.integrator.defineCollections(collections);
 
-      // NOTICE: Check each Smart Action declaration to detect configuration errors.
-      _.each(collections, (collection) => {
-        if (collection.actions) {
-          _.each(collection.actions, (action) => {
-            if (action.fields && !_.isArray(action.fields)) {
+      collections
```

Comment:
[ tests ] Is it possible to add a test for this ?

Reply:
UPDATE : As discussed with @rap2hpoutre it would be painful to add tests for such little changes. We should split this file and add tests later on.